src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -325,23 +325,35 @@ public function importModuleAction(Request $request)
         $moduleManager = $this->get('prestashop.module.manager');
         try {
             $file_uploaded = $request->files->get('file_uploaded');
-            $violations = $this->get('validator')->validate($file_uploaded, new File(
-                [
+            $notNullViolations = $this->get('validator')->validateValue(
+                $file_uploaded,
+                new Assert\NotNull(array(
+                        'message' => $this->get('translator')->trans(
```

Comment:
The component already uses the ``Translator`` [component](http://symfony.com/doc/current/book/translation.html#translating-constraint-messages).

If you want to configure a new translation domain, this is [how you can do it](https://symfony.com/doc/2.8/reference/configuration/framework.html#translation-domain):

```yml
framework:
    translation_domain: "Admin.Modules.Notification"
```

*Be careful*, this will change domain for every error message used by Symfony Validator.

We'd better should create our own "prestashop" validator using the ``translation_domain`` of our choice:

```php
$translator = new Translator('en'); 
$translator->addLoader('xliff', new XliffFileLoader()); // translation format ?
$translator->addResource('xliff', __DIR__ . '/vendor/symfony/validator/Symfony/Component/Validator/Resources/translations/validators.en.xlf', 'en'); // to be changed by the path to the translations of `Admin.Modules.Notification`
 
$validator = Validation::createValidatorBuilder()
    ->setTranslator($translator)
    ->getValidator();
```

What do you think ?